collector/generatorreceiver/topos/hipster_shop.json

```diff
@@ -87,7 +87,7 @@
       },
       {
         "serviceName" : "productcatalogservice",
```

Comment:
Ah this reminds me that I need to update `hipster_shop.json` to use maps instead of lists for `services` and `routes` 😅  I'll make a PR for that (nothing for you to do in this PR, just a heads-up because if you try to run this config right now it will yield a validation error)!